### PR TITLE
YR/-Ews-update-to-latest-version-of-exchangelib/CIAC-10245

### PR DESCRIPTION
**Pull request by RosenbergYehuda**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10245

## Description
This PR offers a fix for an API breaking backward compatibility: updating the Docker image with the latest version of "exchangelib" to resolve the issue. While a previous PR introduced a workaround, this one reverts that change, keeping the beneficial fixes intact. and reducing running time.
Since we require the latest version of exchangelib within the Docker environment, and the native Docker image does not include that version, I've excluded the native image from this package.
this contains revert to workaround implemented in the following PRs:
https://github.com/demisto/content/pull/33771/files
https://github.com/demisto/content/pull/33754/files

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@amshamah419 @thefrieddan1 @JudahSchwartz @GuyAfik @samuelFain @JasBeilin Doc review completed.  Looks OK

**Review by JudahSchwartz — DISMISSED** [review]

File: Packs/MicrosoftExchangeOnline/ReleaseNotes/1_3_4.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### EWS O365
+
+- Updated the Docker image to: *demisto/py3ews:1.0.0.91864*.
+- Improved implementation of the code.
```
Comment (JudahSchwartz) [inline Packs/MicrosoftExchangeOnline/ReleaseNotes/1_3_4.md:7]: better to give a less vague description

File: Tests/docker_native_image_config.json [hunk 2]
```diff
@@ -107,6 +107,15 @@
                 "native:candidate"
             ]
         },
+        {
+            "id":"EWSO365",
+            "reason":"issue: CIAC-10245 - microsoft broke their aoi, and fix it in an newer version of exchangelib package",
```
Comment (JudahSchwartz) [inline Tests/docker_native_image_config.json:112]: a couple typos and inaccurate. Exchangelib isnt maintained by microsoft

File: Packs/MicrosoftExchangeOnline/ReleaseNotes/1_3_4.md (hunk 1 above)
Comment (RosenbergYehuda) [reply Packs/MicrosoftExchangeOnline/ReleaseNotes/1_3_4.md:7]: I'm unsure how to document their changes made to the code; it's simply reverting with no impact on clients. I used something I found in our repository. Any suggestions?

Comment (JasBeilin) [inline Packs/MicrosoftExchangeOnline/ReleaseNotes/1_3_4.md:7]: Maybe this?
```suggestion
- Improved internal infrastructure to reduce runtime.
```

**Review by JasBeilin — APPROVED** [review]
Looks good on my side, I do think we need docs review for the release notes, please wait until @ShirleyDenkberg does a review.

**Pull request merged**